invenio_rdm_records/services/schemas/pids.py

```diff
@@ -14,8 +14,8 @@
 #
 # PIDs
 #
-class PIDSchema(Schema):
-    """PIDs schema."""
+class ExternalPIDSchema(Schema):
```

Comment:
it was me to introduce `External` as first, wondering if we should keep simply `PID` as in the record json schema...

Reply:
Agree! Done :)